app/commands/user/insiders_status/determine_eligibility_status.rb

```diff
@@ -2,7 +2,8 @@ class User::InsidersStatus::DetermineEligibilityStatus
   include Mandate
 
   LIFETIME_REPUTATION_THRESHOLD = 1_000
-  LIFETIME_DONATIONS_THRESHOLD = 500_00
+  LIFETIME_DONATIONS_THRESHOLD = 499_00
+  LIFETIME_PREMIUM_AMOUNT = 499_00
```

Comment:
This should be unified with the constant in `Payments::Stripe::Price` to be just one constant.

Reply:
We should unify, but this is not stripe specific though (it also holds for paypal donations). Any ideas for a better place?